docs/development/pigs/pig-026.rst

```diff
@@ -0,0 +1,328 @@
+.. include:: ../../references.txt
+
+.. _pig-026:
+
+********************************
+PIG 26 - Model Priors API 
+********************************
+
+* Author: Noah Biederbeck, Katrin Streil
+* Created: ``2023-03-13``
+* Accepted:
+* Status: draft
+* Discussion: `#4381`_
+
+Abstract
+========
+
+This PIG is intended to introduce priors on models or on parameters that are evaluated during fitting.
+
+Motivation
+==========
+
+Using priors on models or on parameters is the next step in full-fledged statistical analyses. 
+A prior can incorporate the analysers' knowledge of the topic at hand or information about
+the estimated IRFs systematics provided by the corresponding experiment, and yield more realistic and trustworthy results. 
+The proposed formalism also includes the application of nuisance parameters and regularization.
+
+Use cases
+=========
+
+In the past priors were a regularly requested feature or the solution to a problem.
+See the following issues and PRs of Gammapy:
+
+- Add sigma v estimator functionality for dark matter use case `#2075`_
+- Background systematics as a nuisance parameter `#3955`_
+- Support unfolding methods for spectral flux points `#4122`_
+- Add PiecewiseNormSpatialModel `#4208`_
+
```

Comment:
This is also my main remaining comment. Adding the uses cases in code. E.g. for the unfolding it could look like:

```python3
import numpy as np
from astropy import units as u
from gammapy.modeling.model import PiecewiseNormSpectralModel, MultivariateGaussianPrior, PowerLawSpectralModel

n_points = 10

pwl = PowerLawSpectraModel()

energy = np.geomspace(1, 10, n_points) * u.TeV
norm =  PiecewiseNormSpectralModel(energy=energy)

prior = MultivariateGaussianPrior.from_(means=np.ones(10), covariance_type="diagonal")
norm.prior = prior

spectral_model = pwl * norm
```

Reply:
Thanks @registerrier and @adonath! I will move the use cases up and also add a similar example as you showed here!